app/models/user.rb

```diff
@@ -113,6 +118,21 @@ def traits_for_segment
       demo_user: demo_user?,
       biohub_user: biohub_user?,
       czi_user: czi_user?,
+      # Counts (should be cached in the users table for perf)
+      projects: projects.size, # projects counter is NOT cached because has_and_belongs_to_many
+      samples: samples.size,
+      favorite_projects: favorite_projects.size,
+      favorites: favorites.size,
+      visualizations: visualizations.size,
+      phylo_trees: phylo_trees.size,
+      # Has-some (this is important for Google Custom Dimensions)
+      # See https://segment.com/docs/destinations/google-analytics/#custom-dimensions
+      has_projects: !projects.empty?,
```

Comment:
Do these dimensions have to be boolean or have a specific prefix? 
Sorry, browsed the link but could not find the constraints for google analytics.

Can you add a comment why the sizes above do not suffice for google analytics?